app/controllers/reports_controller.rb

```diff
@@ -2,31 +2,23 @@ class ReportsController < ApplicationController
   include Foreman::Controller::AutoCompleteSearch
 
   before_filter :setup_search_options, :only => :index
+  before_filter :deprecated
```

Comment:
Indeed, just remove this controller as we did with ConfigTemplates.  Also remove its old views.